docs/tutorials/tutorial-a-guide-to-using-the-snowplow-inspector-extension/index.md

```diff
@@ -0,0 +1,173 @@
+---
+title: "A Guide to Using the Snowplow Inspector Extension"
+date: "2021-06-09"
+sidebar_position: 90
+toc_max_heading_level: 4
+---
+
+At Snowplow we frequently use the [Snowplow Inspector chrome extension](https://chrome.google.com/webstore/detail/snowplow-inspector/maplkdomeamdlngconidoefjpogkmljm?hl=en) to check for event emission on a web page - and we also recommend that our clients do the same when in the process of validating their tracking code.
+This extension is an [Open Source project](https://github.com/poplindata/chrome-snowplow-inspector) by [Poplin Data](https://poplindata.com/), one of our partners operating in Australia and New Zealand.
+In 2023, Poplin Data [was acquired by Snowplow](https://snowplow.io/blog/snowplow-acquires-poplin-data/) and became Snowplow Australia.
+
+Once added to Chrome the extension can be opened by [opening Developer Tools](https://developer.chrome.com/docs/devtools/open/) (usually <kbd>Ctrl</kbd>+<kbd>Shift</kbd>+<kbd>I</kbd> or on Mac <kbd>Cmd</kbd>+<kbd>Option</kbd>+<kbd>I</kbd>), where it has its own tab named ‘Snowplow’ (look for the tab adjacent to ‘Elements’, ‘Console’, ‘Network’, etc. - it may be necessary to expand the list of tabs using the chevrons button (»)). 
```

Comment:
```suggestion
Once you add the extension to Chrome, you can view it by [opening Developer Tools](https://developer.chrome.com/docs/devtools/open/) (usually <kbd>Ctrl</kbd>+<kbd>Shift</kbd>+<kbd>I</kbd> or on Mac <kbd>Cmd</kbd>+<kbd>Option</kbd>+<kbd>I</kbd>), where it has its own tab named ‘Snowplow’ (look for the tab adjacent to ‘Elements’, ‘Console’, ‘Network’, etc. - it may be necessary to expand the list of tabs using the chevrons button (»)). 
```

Reply:
I'm a sucker for passive voice. :stuck_out_tongue: I've included all your suggestions.